src/frontmenu_ingame_tabs.c

```diff
@@ -1205,6 +1205,7 @@ void gui_go_to_next_creature_activity(struct GuiButton *gbtn)
 
 RoomIndex find_my_next_room_of_type(RoomKind rkind)
```

Comment:
I see you're modifying an original function here. Why, and isn't there stuff still using this?

Reply:
I needed to be able to specify a player, so made a new function allowing me to do so, and made the original a wrapper.

Reply:
I see.

You can just remove the bit you commented out.